tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -244,50 +244,49 @@ void test_inverse_map(const Map& map,
 
 /*!
  * \ingroup TestingFrameworkGroup
- * \brief Given a 3D Map `map`, tests the map functions, including map inverse,
+ * \brief Given a Map `map`, tests the map functions, including map inverse,
  * jacobian, and inverse jacobian, for a series of points.
  */
 template <typename Map>
-void test_map3d(const Map& map) {
+void test_suite_for_map(const Map& map) {
   // Set up random number generator
   std::random_device rd;
+  const auto seed = rd();
   std::mt19937 gen(rd());
```

Comment:
Since you're assigning `rd()` to `seed` shouldn't you be using `seed` on this line? Then you're seeding the RNG with a different seed than what you're printing out.